test/cpp/run_tests.sh

```diff
@@ -58,7 +58,7 @@ make -j $VERB
 
 if [ $BUILD_ONLY -eq 0 ]; then
   if [ "$LOGFILE" != "" ]; then
-    ./test_ptxla ${FILTER:+"$FILTER"} 2>$LOGFILE
+    ./test_ptxla ${FILTER:+"$FILTER"} 2>> $LOGFILE
```

Comment:
Let's leave this one unchanged. On the development environment we often use `./run_tests.sh` to verify if all test is passing. It is not ideal to accumulate a huge log file over the time.

Reply:
You don't need to rerun the CI for this one.